docs/geedocs/docsrc/answer/geeServerConfigAndSecurity/configureGeeForLDAP.rst

```diff
@@ -93,9 +92,9 @@ Configure GEE Server Admin page for LDAP authentication
                 AuthType Basic
 
                 AuthBasicProvider ldap
-                AuthLDAPBindDN "user@server.name.local"
-                AuthLDAPBindPassword "localuserpassword"
-                AuthLDAPURL "ldap://server.name.local:389/cn=Users,dc=domain,dc=google,dc=com?uid"
+                AuthLDAPBindDN "someuser@example.com"
+                AuthLDAPBindPassword "somepassword"
+                AuthLDAPURL "ldap://ldapserver.example.com:389/ou=Users,dc=example,dc=com?uid"
 
                 <Limit GET>
                 require valid-user
```

Comment:
Can't comment where it belongs, but in AuthLDAPURL below, should be where to look for users, delete at